handlers/bootstrap.go

```diff
@@ -32,6 +31,15 @@ func ActivateMachine(event *events.Event, apiClient *client.RancherClient) (err
 		"eventId":    event.Id,
 	}).Info("Activating Machine")
 
+	if err := acquire_networked_global_persistent_lock(event.ResourceId); err != nil {
+		log.WithFields(log.Fields{
+			"err": err,
+		}).Info("Could not acquire global lock")
+		reply := newReply(event)
+		return publishReply(reply, apiClient)
```

Comment:
empty reply makes cattle think that it was successful. Not replying is the right thing to do.